### DISCUSSION TITLE
HealthCheck 是否支持通过yaml 配置？

### DISCUSSION THREAD

**Discussion by ztl8702** [Serverless-Devs/Serverless-Devs#628] [Q&A] [answered]
函数计算控制台可以设置health check

<img width="1324" alt="Screenshot 2023-01-04 at 3 47 22 pm" src="https://user-images.githubusercontent.com/2714446/210486377-c5ad7e61-064b-42b9-8030-448118a89a7b.png">

但是 [yaml 规范](https://docs.serverless-devs.com/fc/yaml/readme) 里面好像没有找到

**Comment by wss-git** [comment, marked as answer]
https://github.com/devsapp/fc/issues/834

还没有支持，后面我们会高优排期进行支持
